client/src/commands/run.ts

```diff
@@ -86,7 +86,6 @@ function _run(selected = false) {
 
   runCode(selected)
     .catch((err) => {
-      console.dir(err);
```

Comment:
Looking through the code, it looks like this was probably intentional. Can you confirm, @scnwwu?

Reply:
Yes, it would be helpful to see the entire error object for trouble shooting.